### PR TITLE
docs(contributing): init contributing doc

### PR DESCRIPTION
**Pull request by Slashgear**

### CODE REVIEW

File: docs/contributing.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
+# Contributing
+
+Open blog is before all an open source project to share a publication of blog posts with the community.
+
+> Developed by the community for everyone freely
+
+## Share your blog posts :rocket:
+
+Before adding your blog post to Open blog, you should check those 3 steps.
```
Comment (mfrachet) [inline docs/contributing.md]: those => these (those à un sens plus péjoratif)

File: docs/contributing.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+### Post rules
+
+- Post should follow the [Code of Conduct](https://github.com/open-blog-initiative/open-blog.dev/blob/master/CODE_OF_CONDUCT.md).
```
Comment (mfrachet) [inline docs/contributing.md:13]: Add the before post would make sense nop ? 🤔

File: docs/contributing.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+- Post can't be direct advertising.
+- Post can't break any copyright in its content.
+- Post should not be published in another blog.
```
Comment (mfrachet) [inline docs/contributing.md]: ???

File: docs/contributing.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+- Post should be written in one of those languages:
```
Comment (mfrachet) [inline docs/contributing.md]: those => these

File: docs/contributing.md [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+  - French :fr:
+  - English :gb:
+
+NB: For now we only support two languages because those languages are maintainer ones.
```
Comment (mfrachet) [inline docs/contributing.md]: maintainer est utilisé plusieurs fois dans cette phrase

File: docs/contributing.md [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+
+or
+
+```sh
+yarn docs:dev
+```
+
+#### Create your post file
+
+In `docs/post` folder, create a folder with your github name.
```
Comment (mfrachet) [inline docs/contributing.md:59]: On devrait peut être faire un super simple cli pour générer un nouvel article?

File: docs/contributing.md [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+In this fresh new directory, just create a [Markdown file](https://fr.wikipedia.org/wiki/Markdown).
+Name it with a dashCase title (example: `my-awesome-post.md`.
+
+If you don't know how to write a Markdown file matching Vuepress rules, check the [documentation](https://v1.vuepress.vuejs.org/guide/markdown.html).
+
+##### Metadata
+
+At the beginning of you blog post, you must define `frontmatter` block to add metadata to you post.
```
Comment (mfrachet) [inline docs/contributing.md]: to youR post

File: docs/contributing.md [hunk 8]
```diff
@@ -0,0 +1,112 @@
...
+
+```yml
+------
+type: post
+author: Antoine CARON # Required
+pseudo: Slashgear # If you have one
+profile_picture: https://my-awesome-image/profile.jpg # Only external on, twitter image for example.
```
Comment (mfrachet) [inline docs/contributing.md]:
```diff
-Only external on, twitter image for example.
+Only external one, twitter image for example.
```

File: docs/contributing.md [hunk 9]
```diff
@@ -0,0 +1,112 @@
...
+biography: Bla bla bla about you # truncate after 280 characters.
```
Comment (mfrachet) [inline docs/contributing.md]: truncates ou truncated

File: docs/contributing.md [hunk 10]
```diff
@@ -0,0 +1,112 @@
...
+github_profile: Slashgear # will display github icon link to you profile
+twitter_profile: Slashgear_ # will display twitter icon link to you profile
+meta: # SEO meta
+  - name: og-title
+    content: Title example
```
Comment (mfrachet) [inline docs/contributing.md:80]: Not missing a `-` here?

File: docs/contributing.md [hunk 11]
```diff
@@ -0,0 +1,112 @@
...
+---
+
+```
+
+##### Author Badge
+
+If you want to display a nice author presentation block, make sure to define metadata in the `frontmatter` part.
+
+Then add at the end of your post.
```
Comment (mfrachet) [inline docs/contributing.md]: Then add at the end of your post: (pas de point àla fin)

File: docs/contributing.md [hunk 12]
```diff
@@ -0,0 +1,112 @@
...
+
+```html
+<Author />
+```
+
+##### Comments section
+
+If you want to let user comment your blog post, just add this add the end of your post.
```
Comment (mfrachet) [inline docs/contributing.md]: same as previous?

File: docs/contributing.md [hunk 13]
```diff
@@ -0,0 +1,112 @@
...
+
+```html
```
Comment (mfrachet) [inline docs/contributing.md]: I think JSX is a valid markdown attr

File: docs/contributing.md [hunk 14]
```diff
@@ -0,0 +1,112 @@
...
+
+### Open your pull request
+
+:construction:
+
+Pre-reading, typos, linter, PR state, Review time.
+
+## Make "Open Blog" a better place :sparkles:
+
+If you have any idea to improve _Open blog_ globally, don't hesitate to open a discussion [Github issues](https://github.com/open-blog-initiative/open-blog.dev/issues/new) (Take 5 minutes to check if your idea have not been submitted yet).
```
Comment (mfrachet) [inline docs/contributing.md]: your idea have => has

File: docs/contributing.md [hunk 15]
```diff
@@ -0,0 +1,112 @@
...
+
+This publication is based on [Vuepress](https://v1.vuepress.vuejs.org/), your idea could also be a great contribution to this project.
```
Comment (mfrachet) [inline docs/contributing.md]: ideaS

File: docs/contributing.md (hunk 2 above)
Comment (Slashgear) [reply docs/contributing.md:13]: What do you mean ?

File: docs/contributing.md (hunk 3 above)
Comment (Slashgear) [reply docs/contributing.md]: SEO purpose my friend. If you publish exactly the same content in many page. Google won't understand and will say "Duplicate content" 

Here is a solution https://www.yakaferci.com/link-rel-canonical/

File: docs/contributing.md (hunk 13 above)
Comment (Slashgear) [reply docs/contributing.md]: Yep but here it is not JSX

File: docs/contributing.md (hunk 6 above)
Comment (Slashgear) [reply docs/contributing.md:59]: Yep, if you want, go for it 💡 or open a issue at least 😉

File: docs/contributing.md (hunk 10 above)
Comment (Slashgear) [reply docs/contributing.md:80]: Nop, Yml for the win 

Here it is like 
```json
{
  "meta": [
    {
      "name": "og-title",
      "content": "Title example"
    }
  ]
}

File: docs/contributing.md (hunk 3 above)
Comment (mfrachet) [reply docs/contributing.md]: I don't really agree that SEO should take precedence over freedom 🤔 . I would like to publish my post everywhere I want 😛

File: docs/contributing.md (hunk 6 above)
Comment (mfrachet) [reply docs/contributing.md:59]: I'll take a look on friday :)

**Review by mfrachet — APPROVED** [review]

**Pull request merged**